### ISSUE TITLE
[Bug][Linux] Wrong action on click file above expandable folder 

### ISSUE THREAD

**Issue by AmirHosseinKarimi** [atom/atom#19804]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: <https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode>
    * Followed all applicable steps in the debugging guide: <https://flight-manual.atom.io/hacking-atom/sections/debugging/>
    * Checked the FAQs on the message board for common solutions: <https://discuss.atom.io/c/faq>
    * Checked that your issue isn't already filed: <https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom>
    * Checked that there is not already an Atom package that provides the described functionality: <https://atom.io/packages>

Please look this video : https://gofile.io/?c=Y5AZ3A

When click on expandable folder and then click on a file above the folder that will expand and collapse the folder again and does not open the file, same as when click on folder not the file. 

### Steps to Reproduce

1. Click on a folder to expand / collapse / highlight
2. Click on a folder immediatly above the folder

**Expected behavior:**

That should open file preview

**Actual behavior:**

That will expand/collapse the folder that before clicked

**Reproduces how often:**

100%

### Versions

Atom    : 1.38.2
Electron: 2.0.18
Chrome  : 61.0.3163.100
Node    : 8.9.3
OS : Linux, Debian 10

### Additional Information

Screen record to reproduce the problem : https://gofile.io/?c=Y5AZ3A

**Comment by rsese** [comment]
Yeah that's just a workaround - since the One themes are in the main atom/atom repo I transferred the issue over.
